brownie-config.yaml

```diff
@@ -0,0 +1,66 @@
+project_structure:
```

Comment:
Oh one other thing: can we put this file in the `config/` folder and specify it’s location via a flag somehow? Not a big deal if we can’t but want to keep the root level uncluttered where possible

Reply:
oh yeah i was thinking about that too, but i haven't been able to do it until now

this is from the [documentation](https://eth-brownie.readthedocs.io/en/stable/config.html#config-reports)

> The configuration file must be saved as brownie-config.yaml. If saved in the root directory of a project it will be loaded whenever that project is active. If saved in your [home path](https://docs.python.org/3/library/pathlib.html#pathlib.Path.home), it will always be loaded.


i'll keep trying, though